src/services/videoconference/index.js

```diff
@@ -507,13 +497,121 @@ class CreateVideoconferenceService {
 	}
 }
 
+const jwtVerify = promisify(jwt.verify);
+
+class RecordingReadyVideoconferenceService {
+	constructor(app) {
+		this.app = app;
+		this.docs = {};
+	}
+
+	async create(data, params) {
+		const { route } = params;
+		const { meeting_id: meetingId, record_id: recordId } = await jwtVerify(data.signed_parameters, SALT);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['meetingId' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5485544)